{
	"id": "oval:com.oracle.elsa:def:20243846",
	"advisories": [
		{
			"id": "oval:com.oracle.elsa:def:20243846",
			"title": "\nELSA-2024-3846:  python-idna security update (MODERATE)\n",
			"description": "\n[2.10-7.0.1.1]\n- Rebuild with release bump\n\n[2.10-7.1]\n- Security fix for CVE-2024-3651\nResolves: RHEL-33464\n",
			"severity": [
				{
					"type": "vendor",
					"source": "linux.oracle.com/security",
					"vendor": "MODERATE"
				}
			],
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2024-3651.html"
				},
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/errata/ELSA-2024-3846.html"
				}
			],
			"published": "2024-06-13T00:00:00Z"
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2024-3651",
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2024-3651.html"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "oracle:9",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "python3-idna",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:2.10-7.0.1.el9_4.1"
								}
							],
							"fixed": [
								"0:2.10-7.0.1.el9_4.1"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "oracle"
}
